src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -61,6 +63,11 @@
       <AssemblyInfoLines Condition="'$(SkipFrameworkAssemblyMetadata)' != 'true'" 
         Include="[assembly:System.Reflection.AssemblyMetadata(&quot;%(AssemblyMetadata.Identity)&quot;, &quot;%(AssemblyMetadata.Value)&quot;)]" />
     </ItemGroup>
+    
+    <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true' and '$(DisableTests)' != 'true'">
```

Comment:
Let not add this in our general versioning.targets file. Please include this in the perftesting.targets, that is why we have this extensibility point so people can add to it outside this file.